src/core/client.ts

```diff
@@ -128,11 +128,7 @@ export interface ApiClientOptions {
      * @param method The API method to be called, e.g. `getMe`
      * @returns The URL that will be fetched during the API call
      */
-    buildUrl?: (
-        root: string,
-        token: string,
-        method: string,
-    ) => Parameters<typeof fetch>[0];
+    buildUrl?: (root: string, token: string, method: string) => string;
```

Comment:
```suggestion
    buildUrl?: (root: string, token: string, method: string) => string | URL;
```
as discussed in https://t.me/grammyjs/41240.

Note that URL must be imported from platform-specific modules.

Reply:
Correction: this is no longer the case, I was remembering things from before Node 10.